front/src/main/kotlin/com/sns/front/core/security/WebfluxSecurityConfig.kt

```diff
@@ -0,0 +1,108 @@
+package com.sns.front.core.security
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.sns.front.utils.JsonUtil
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Bean
+import org.springframework.security.config.Customizer.withDefaults
+import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity
+import org.springframework.security.config.web.server.ServerHttpSecurity
+import org.springframework.security.config.web.server.ServerHttpSecurity.AuthorizeExchangeSpec
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.authority.SimpleGrantedAuthority
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserRequest
+import org.springframework.security.oauth2.client.userinfo.ReactiveOAuth2UserService
+import org.springframework.security.oauth2.core.user.DefaultOAuth2User
+import org.springframework.security.oauth2.core.user.OAuth2User
+import org.springframework.security.web.server.SecurityWebFilterChain
+import org.springframework.security.web.server.authentication.logout.RedirectServerLogoutSuccessHandler
+import org.springframework.stereotype.Service
+import org.thymeleaf.extras.springsecurity5.dialect.SpringSecurityDialect
+import reactor.core.publisher.Mono
+import java.net.URI
+import java.util.*
+
+/**
+ * @author Hyounglin Jun
+ */
+@EnableWebFluxSecurity
+class WebfluxSecurityConfig {
+
+    @Bean
+    fun securityDialect(): SpringSecurityDialect? {
+        return SpringSecurityDialect()
+    }
+
+    @Bean
+    fun springSecurityFilterChain(http: ServerHttpSecurity): SecurityWebFilterChain? {
+        val redirectServerLogoutSuccessHandler = getRedirectServerLogoutSuccessHandler()
+
+        return http
+            .authorizeExchange { exchanges: AuthorizeExchangeSpec ->
+                exchanges
+                    .pathMatchers("/", "/home", "/auth-api/**", "/register", "/js/**", "/css/**", "/user-api/**").permitAll()
```

Comment:
```suggestion
            .authorizeExchange {
                it
                    .pathMatchers("/", "/home", "/auth-api/**", "/register", "/js/**", "/css/**", "/user-api/**").permitAll()
```
혹시나 코멘트드리면 자바랑 다르게 타입이랑 명시하지 않아도 됩니다